sdk/core/azure-core-tracing-opentelemetry/src/main/java/com/azure/core/tracing/opentelemetry/OpenTelemetryUtils.java

```diff
@@ -130,20 +130,22 @@ static void addAttribute(Span span, String key, Object value, OpenTelemetrySchem
     }
 
     /**
-     * Parses an OpenTelemetry Status from AMQP Error Condition.
+     * Parses an OpenTelemetry status from error description.
      *
      * @param span the span to set the status for.
-     * @param statusMessage description for this error condition.
+     * @param statusMessage description for this error condition. Any non-null {@code statusMessage} indicates an error.
+     *                      Pass empty string or {@code "error"} string to create status without description.
      * @param throwable the error occurred during response transmission (optional).
      * @return the corresponding OpenTelemetry {@link Span}.
      */
-    public static Span setError(Span span, String statusMessage, Throwable throwable) {
+    static Span setError(Span span, String statusMessage, Throwable throwable) {
         if (throwable != null) {
             span.recordException(throwable);
             return span.setStatus(StatusCode.ERROR, statusMessage);
         }
 
-        if (statusMessage != null) {
+        // "success" is needed for back compat with older Event Hubs and Service Bus, don't use it.
+        if (statusMessage != null && !"success".equals(statusMessage)) {
```

Comment:
This will strictly be `success`, right? No chances of `Success` or `SUCCESS`?

Reply:
no, only "success" and only in the past